src/services/item/plugins/geolocation/service.ts

```diff
@@ -78,14 +78,17 @@ export class ItemGeolocationService {
     // TODO optimize?
     return geoloc
       .map((g) => {
-        if (g.item.id in itemMemberships.data) {
+        const itemId = g.item.id;
+        if (
+          // accessible items
```

Comment:
Maybe extracting the conditions in well named variables allow you to avoid comments ?

`const itemIsAccessible = itemId in itemMemberships.data;` or a better name ;)